skema/text_reading/text_reading/webapp/conf/application.conf

```diff
@@ -15,6 +15,10 @@ play.filters {
   disabled += "play.filters.csrf.CSRFFilter"
    enabled += "play.filters.csp.CSPFilter"
 }
+
+skema.hostname = "skema.clulab.org"
```

Comment:
Can this be parameterized for running the web app either locally directly with docker or via a docker-compose file?

Reply:
The next line down, `skema.hostname = ${?SKEMA_HOSTNAME}`, allows the value to be taken from an environment variable.  The clulab.org value is in effect a default.  So, I think this one should be OK.

Reply:
Nice!